src/PrestaShopBundle/Resources/config/services/bundle/common.yml

```diff
@@ -0,0 +1,17 @@
+# Common service from bundle that are needed in Admin and in Front (like some Entity repositories)
+services:
+  _defaults:
+    public: true
+
+  PrestaShopBundle\Entity\Repository\FeatureFlagRepository:
+    factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
+    arguments:
+      - PrestaShopBundle\Entity\FeatureFlag
+    lazy: true
+
+  PrestaShopBundle\Entity\Repository\TabRepository:
+    factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
+    public: false
+    arguments:
+      - PrestaShopBundle\Entity\Tab
+    lazy: true
```

Comment:
```suggestion
services:
  _defaults:

  PrestaShopBundle\Entity\Repository\FeatureFlagRepository:
    factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
    public: true
    arguments:
      - PrestaShopBundle\Entity\FeatureFlag
    lazy: true

  PrestaShopBundle\Entity\Repository\TabRepository:
    factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
    arguments:
      - PrestaShopBundle\Entity\Tab
    lazy: true
```

Reply:
(is the public necessary?)

Reply:
Good question, I'll check but maybe a wrong copy/paster indeed

Reply:
Seems like they can, and should, be private

Reply:
They are private now